src/styles/blocks/style-block-header.scss

```diff
@@ -0,0 +1,15 @@
+.header {
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-link {
+    display: block;
+    width: 40px;
+    height: 40px;
+    @include transition (transform, 300ms);
```

Comment:
It is recommended to use variables for commonly used values like duration (300ms) to maintain consistency across your codebase.